src/lib/coerce.js

```diff
@@ -309,3 +335,22 @@ exports.coerceFont = function(coerce, attr, dfltObj) {
 
     return out;
 };
+
+exports.validate = function(value, opts) {
+    var valObject = exports.valObjects[opts.valType];
+
+    if(opts.arrayOk && Array.isArray(value)) return true;
+
+    if(valObject.validateFunction) {
+        return valObject.validateFunction(value, opts);
+    }
+
+    var failed = {},
+        out = failed,
+        propMock = { set: function(v) { out = v; } };
+
+    // 'failed' just something mutable that won't be === anything else
+
+    valObject.coerceFunction(value, propMock, failed, opts);
```

Comment:
No. Because we want coerced values to be considered valid.

For example,

```js
Lib.validate('1', { valType: 'number' });  // => true
```

In other words, `out` and `failed` will be linked to different refs when the coerce function makes `propMock` execute `propMock.set(dflt)`.